ansible/roles/oracle-db-monitoring/tasks/install-oracle-db-monitoring.yml

```diff
@@ -22,9 +26,26 @@
   loop_control:
     index_var: index
   notify: reload script-exporter
+  when: oracle_monitoring_list is defined
+
+- name: Install script to be run by script exported to monitor connection to fixngo
+  template:
+    src: fixngo-connection.sh.j2
+    dest: "/opt/script-exporter/fixngo-connection.sh"
+    mode: 0500
+    owner: oracle
+  notify: reload script-exporter
+  when: fixngo_connection_target is defined
 
 - name: Start & Enable Service
   service:
     name: script-exporter
     state: started
     enabled: yes
+
+# added a reload to make sure config is loaded
+- name: reload script-exporter
```

Comment:
You would normally have this as a handler, e.g. if config changed, trigger a handler to restart the service

Reply:
There is a handler for the config change AND a service check but having logged onto the machine after this has run it only picks up the changes following another service restart